common/src/main/java/whocraft/tardis_refined/common/tardis/manager/TardisExteriorManager.java

```diff
@@ -176,6 +158,9 @@ public void placeExteriorBlock(TardisLevelOperator operator, TardisNavLocation l
         ResourceLocation theme = (aestheticHandler.getShellTheme() != null) ? aestheticHandler.getShellTheme() : ShellTheme.HALF_BAKED.getId();
         ShellTheme shellTheme = ShellTheme.getShellTheme(theme);
 
+        //remove the exterior block
+        location.getLevel().setBlock(location.getPosition(), Blocks.AIR.defaultBlockState(), Block.UPDATE_ALL);
```

Comment:
Use of specific method ``removeBlock`` may be better user here to be consistent with other instances

Reply:
We actually favor setblock over the remove block for most of the codebase.